frontends/election-manager/src/screens/manual_data_import_index_screen.test.tsx

```diff
@@ -157,7 +157,7 @@ test('precinct table renders properly when there is no data', () => {
   );
 });
 
-test('loads prexisting manual data to edit', async () => {
+test('loads preexisting manual data to edit', async () => {
```

Comment:
Also possibly:
```suggestion
test('loads existing manual data to edit', async () => {
```

Or `existing precinct manual data...`